README.md

```diff
@@ -49,9 +49,10 @@ $ stack build   # Compile Courseography and all Haskell dependencies (this will
 #### Parsing and Generation
 
 1. Create the database file `$ stack run database-setup`
-2. Parse prerequisite graphs `$ stack run database-graphs`
-3. Parse course information from Arts and Science Calendar `$ stack run database-calendar`
-4. Parse course information from Arts and Science Timetable `$ stack run database-timetable`
+2. Run necessary database migrations `$ stack run database-migrate`
```

Comment:
Similar to my comment below, I don't think this is necessary. If the database is being created for the first time, it should be sufficient to just run the `migrateAll` to get the current version of the schema.